src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XContainer.cs

```diff
@@ -979,6 +979,65 @@ public bool ReadContentFrom(XContainer rootContainer, XmlReader r)
                 return true;
             }
 
+            public async ValueTask<bool> ReadContentFromAsync(XContainer rootContainer, XmlReader r)
```

Comment:
```suggestion
            private async ValueTask<bool> ReadContentFromAsync(XContainer rootContainer, XmlReader r)
```

Reply:
Be careful, this must be a public method of the `ContentReader` private nested class in order to be used by its `XContainer` parent class.

Reply:
I see, could be internal too, but as its private class doesn't matter, no need to update